docker/rucio_client/scripts/setSiteAvailability

```diff
@@ -5,12 +5,14 @@ import json
 import os
 import requests
 import time
-
```

Comment:
Please leave this blank line. It's good to separate standard libs from our own

Reply:
done